modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/thunks/pageLanguageUpdate.es.js

```diff
@@ -49,7 +50,10 @@ const formatDataRecord = (languageId, pages, preserveValue) => {
 						[languageId]: [],
 					};
 				}
-				else {
+				else if (
+					localizedValue[languageId] ||
```

Comment:
Hey @marcosapmf , check this validation

```localizedValue[languageId] || (localizedValueEdited && localizedValueEdited[languageId])```

Is also repeated in another ```else``` ( line 67 )

This validation could it be a variable and be reused in following lines? 53 and 67

Reply:
Thanks @kevenleone, I'll make this change and then make a `push -f`